src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.spec.ts

```diff
@@ -38,20 +36,11 @@ describe('NfsFormComponent', () => {
         useValue: new ActivatedRouteStub({ cluster_id: undefined, export_id: undefined })
       },
       i18nProviders,
-      SummaryService,
       CephReleaseNamePipe
```

Comment:
Just out of curiosity: can you tell why you removed `CephReleaseNamePipe` in master but not in the backport?

Reply:
Oops, I missed that. Thanks!